core/components/atoms/dropdown/__stories__/variants/Align.story.jsx

```diff
@@ -3,16 +3,17 @@ import Dropdown from '../../Dropdown';
 import { Uncontrolled, Controlled } from '../_common_/types';
 import Text from '@/components/atoms/text';
 import { storyOptions } from '../Options';
+import '../style.css';
 
 // CSF format story
 export const align = () => {
   const dropdownAlignments = ['right', 'left'];
 
   return (
-    <div style={{ display: 'flex', minHeight: '280px' }}>
+    <div className="d-flex Dropdown-wrapper--min">
       {dropdownAlignments.map((alignment, ind) => {
         return (
-          <div key={ind} style={{ marginRight: '20%' }}>
+          <div key={ind} className="Dropdown-spacing">
```

Comment:
Please remove this classes these does not seems to make any difference

Reply:
It does make a difference it defines the spacing between the dropdowns if this is not there the dropdowns will overlap